doc/guide/hive-default.md

```diff
@@ -1 +1,56 @@
-# hive.default
+---
+comment: true
+title: hive.default
+description: Deduplicate options with default node configuration.
+---
+
+# `{{ $frontmatter.title }}`
+
+{{ $frontmatter.description }}
+
+## Introduction
+
+At the top level of a hive wire reserves the `default` attribute. Its applied
+to every node.
+
+::: warning
+
+The `default` configuration must be a "pure configuration", while a node may be
+"impure".
+
+That is to say, `default` must not rely on modules that a
+node imports, but a node may rely on modules that default imports.
```

Comment:
wdym pure configuration whereas a node can be impure? that doesn't make sense to me, also I would omit the last part, or if you still desire to keep this, reword it to be something like

```
`defaults` is imported alongside every defined node in a hive
```

Reply:
I made it easier to read now